src/Utils/dbConnection.ts

```diff
@@ -0,0 +1,31 @@
+import "reflect-metadata";
+import { createConnection } from "typeorm";
+const config = require("./config");
+export const createConn = async () => {
+  try {
+    if (config.NODE_ENV === "test") {
+      return createConnection({
+        type: config.DB_TYPE,
+        host: config.HOST,
+        database: config.TEST_DB,
+        port: config.DB_PORT,
+        username: config.DB_USER,
+        password: config.DB_PASSWORD,
+        entities: ["dist/models/*.js"],
+        migrations: ["dist/migration/*.js"],
+        logging: false,
+        synchronize: false,
+        migrationsRun: true,
+        cli: {
+          migrationsDir: "src/migration",
+          entitiesDir: "src/models"
+        },
+        extra: {
+          connectionLimit: 20
+        }
+      });
+    } else {
+      return createConnection();
+    }
+  } catch (error) {}
```

Comment:
minor thing, could you log the error to the console so that it's known what happened, e.g
```suggestion
catch(error) {
 console.log(`Error creating a connection to the database: ${error}`)
}
```

Reply:
WIP, thanks